src/storage/hummock_sdk/src/lib.rs

```diff
@@ -355,3 +355,25 @@ impl EpochWithGap {
         self.0 & EPOCH_SPILL_TIME_MASK
     }
 }
+
+pub trait ProtoSerializeSizeEstimatedExt {
+    fn estimated_encode_len(&self) -> usize;
+}
+
+pub trait ProtoSerializeExt {
```

Comment:
What's the difference between these two traits? May add some comments to describe the difference.

Reply:
If all appearance of `T` is `Self`, we don't need the `T`, and we can use `Self` directly.

Reply:
PTAL